config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/BetterThanDuctTa-AAAAAAAAAAAAAAAAAAAC8w==.json

```diff
@@ -48,7 +48,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Using a soldering iron tool to remove the \"burned out circuits\" message is cheaper than duct tape. \n\nThe soldering iron requires one soldering material item and 10,000 EU. Valid soldering materials are fine wires, rods, and ingots made from either tin, lead, or soldering alloy. The soldering iron will consume the first such material it finds in the player\u0027s inventory. Since you\u0027ll likely have more than enough, Fine Wires of Soldering Alloy are probably best.\n\nYou can also use it to adjust the redstone strength from weak to strong emitted by GT machines.\n\nAlso, while holding it all pipes, wires, and cables will look very small, so that you can see everything past them, and also be sure of the exact connections.\n\nIn order to \"complete\" the quest you\u0027ll need to charge the soldering iron after you craft it.\n\nFor the other problems other tools are necessary.\n\nPossible problems                Tool to fix\n\"Pipe is loose.\"                    Wrench\n\"Screws are loose.\"           Screwdriver \n\"Something is stuck.\"            Soft Mallet \n\"Platings are dented.\"           Hammer \n\"Circuitry burned out.\"    Soldering Iron \n\"That doesn\u0027t belong there.\"   Crowbar"
+      "desc:8": "Using a soldering iron tool to remove the \"burned out circuits\" message is cheaper than duct tape. \n\nThe soldering iron requires one soldering material item and 10,000 EU. Valid soldering materials are fine wires, rods, and ingots made from either tin, lead, or soldering alloy. The soldering iron will consume the first such material it finds in the player\u0027s inventory. Since you\u0027ll likely have more than enough, Fine Wires of Soldering Alloy are probably best.\n\nYou can also use it to adjust the redstone strength from weak to strong emitted by GT machines.\n\nAlso, while holding it all pipes, wires, and cables will look very small, so that you can see everything past them, and also be sure of the exact connections.\n\nIn order to \"complete\" the quest you\u0027ll need to charge the soldering iron after you craft it.\n\nFor the other problems other tools are necessary. Keep in mind that you can put all needed tools in a toolbox to fix all problems in one click.\n\nPossible problems                Tool to fix\n\"Pipe is loose.\"                    Wrench\n\"Screws are loose.\"           Screwdriver \n\"Something is stuck.\"            Soft Mallet \n\"Platings are dented.\"           Hammer \n\"Circuitry burned out.\"    Soldering Iron \n\"That doesn\u0027t belong there.\"   Crowbar"
```

Comment:
I can see how it might look like a unnecessary duplicate.
However, my reasoning was as follows:
The quest is about maintenance. The toolbox is the tool for maintenance.   
Even if the toolbox has its own quest, it's fitting to mention that toolbox addresses all problems in the quest about maintenance.   
This way, we can prevent the case when a player starts to address the maintenance without the knowledge of the toolbox, wasting their time.

Reply:
ok